### PR TITLE
Update Japanese translation

### PR DESCRIPTION
**Pull request by YuikoTakada**
This is work-in-progress for reviews by Japanese developers.

### CODE REVIEW

File: translations/ja.json [hunk 1]
```diff
@@ -347,7 +347,7 @@
 	"Ignoring invalid pair entry {{.pair}}": "無効なペアエントリー {{.pair}} を無視しています",
 	"Ignoring unknown custom image {{.name}}": "未知のカスタムイメージ {{.name}} を無視しています",
 	"Ignoring unknown custom registry {{.name}}": "未知のカスタムレジストリー {{.name}} を無視しています",
-	"Image was not built for the current minikube version. To resolve this you can delete and recreate your minikube cluster using the latest images. Expected minikube version: {{.imageMinikubeVersion}} -\u003e Actual minikube version: {{.minikubeVersion}}": "",
+	"Image was not built for the current minikube version. To resolve this you can delete and recreate your minikube cluster using the latest images. Expected minikube version: {{.imageMinikubeVersion}} -\u003e Actual minikube version: {{.minikubeVersion}}": "イメージが最新のminikubeバージョンでビルドされていません。minikubeクラスターを削除後、最新のイメージを使用してクラスターを再作成することでこの問題を解決することができます。想定されたminikubeのバージョン:  {{.imageMinikubeVersion}} -\u003e 実際のminikubeのバージョン: {{.minikubeVersion}}",
```
Comment (atoato88) [inline translations/ja.json:350]:
```suggestion
	"Image was not built for the current minikube version. To resolve this you can delete and recreate your minikube cluster using the latest images. Expected minikube version: {{.imageMinikubeVersion}} -\u003e Actual minikube version: {{.minikubeVersion}}": "イメージが現在のminikubeバージョンでビルドされていません。minikubeクラスターを削除後、最新のイメージを使用してクラスターを再作成することでこの問題を解決することができます。想定されたminikubeのバージョン:  {{.imageMinikubeVersion}} -\u003e 実際のminikubeのバージョン: {{.minikubeVersion}}",
```

I think `現在の` is better for `current` on Japanese translation.

File: translations/ja.json [hunk 2]
```diff
@@ -751,7 +751,7 @@
 	"Unable to load profile: {{.error}}": "プロファイルを読み込めません: {{.error}}",
 	"Unable to parse \"{{.kubernetes_version}}\": {{.error}}": "「{{.kubernetes_version}}」を解析できません: {{.error}}",
 	"Unable to parse memory '{{.memory}}': {{.error}}": "メモリー '{{.memory}}' を解析できません: {{.error}}",
-	"Unable to parse version.json: {{.error}}, json: {{.json}}": "",
+	"Unable to parse version.json: {{.error}}, json: {{.json}}": "version.jsonを解析できません: {{.error}}, json: {{.json}}",
```
Comment (atoato88) [inline translations/ja.json:754]:
```suggestion
	"Unable to parse version.json: {{.error}}, json: {{.json}}": "version.json を解析できません: {{.error}}, json: {{.json}}",
```
We need extra space?

File: translations/ja.json [hunk 3]
```diff
@@ -827,7 +827,7 @@
...
 	"You are trying to run a windows .exe binary inside WSL. For better integration please use a Linux binary instead (Download at https://minikube.sigs.k8s.io/docs/start/.). Otherwise if you still want to do this, you can do it using --force": "WSL 内で Windows の .exe バイナリーを実行しようとしています。これより優れた統合として、Linux バイナリーを代わりに使用してください (https://minikube.sigs.k8s.io/docs/start/ でダウンロードしてください)。そうではなく、引き続きこのバイナリーを使用したい場合、--force オプションを使用してください",
 	"You are trying to run the amd64 binary on an M1 system.\nPlease consider running the darwin/arm64 binary instead.\nDownload at {{.url}}": "M1 システム上で amd64 バイナリーを実行しようとしています。\ndarwin/arm64 バイナリーを代わりに実行することをご検討ください。\n{{.url}} でダウンロードしてください。",
-	"You are using the QEMU driver without a dedicated network, which doesn't support `minikube service` \u0026 `minikube tunnel` commands.": "",
+	"You are using the QEMU driver without a dedicated network, which doesn't support `minikube service` \u0026 `minikube tunnel` commands.": "独立したネットワークなしでQEMUドライバーを使用しています。これは`minikube service` \u0026 `minikube tunnel` コマンドをサポートしていません。",
```
Comment (atoato88) [inline translations/ja.json:830]:
```suggestion
	"You are using the QEMU driver without a dedicated network, which doesn't support `minikube service` \u0026 `minikube tunnel` commands.": "独立したネットワークなしで QEMU ドライバーを使用しています。これは`minikube service` \u0026 `minikube tunnel` コマンドをサポートしていません。",
```
ditto.

File: translations/ja.json [hunk 4]
```diff
@@ -913,7 +913,7 @@
...
 	"minikube is unable to connect to the VM: {{.error}}\n\n\tThis is likely due to one of two reasons:\n\n\t- VPN or firewall interference\n\t- {{.hypervisor}} network configuration issue\n\n\tSuggested workarounds:\n\n\t- Disable your local VPN or firewall software\n\t- Configure your local VPN or firewall to allow access to {{.ip}}\n\t- Restart or reinstall {{.hypervisor}}\n\t- Use an alternative --vm-driver\n\t- Use --force to override this connectivity check\n\t": "minikube が VM に接続できません: {{.error}}\n\n\t考えられる理由は以下の 2 つです:\n\n\t- VPN またはファイアウォールによる干渉\n\t- {{.hypervisor}} のネットワーク設定の問題\n\n\t回避策には以下があります:\n\n\t- ローカルの VPN またはファイアウォールを無効化\n\t- {{.ip}} へのアクセスを許可するようにローカルの VPN またはファイアウォールを設定\n\t- {{.hypervisor}} を再起動または再インストール\n\t- 代わりの --vm-driver を使用\n\t- --force を使用してこの接続チェックを上書き\n\t",
-	"minikube mount is not currently implemented with the user network on QEMU": "",
+	"minikube mount is not currently implemented with the user network on QEMU": "minikubeトンネルは、QEMU上のユーザーネットワークでは実装されていません。",
```
Comment (atoato88) [inline translations/ja.json:916]:
```suggestion
	"minikube mount is not currently implemented with the user network on QEMU": "minikube トンネルは、QEMU 上のユーザーネットワークでは実装されていません。",
```
ditto.

**Review by atoato88 — COMMENTED** [review]
Please check comments

File: translations/ja.json (hunk 1 above)
Comment (YuikoTakada) [reply translations/ja.json:350]: done

File: translations/ja.json (hunk 2 above)
Comment (YuikoTakada) [reply translations/ja.json:754]: done

File: translations/ja.json (hunk 3 above)
Comment (YuikoTakada) [reply translations/ja.json:830]: done

File: translations/ja.json (hunk 4 above)
Comment (YuikoTakada) [reply translations/ja.json:916]: done

**Comment by YuikoTakada** [comment]
@atoato88 thank you for your comments. I fixed them, PTAL

File: translations/ja.json [hunk 5]
```diff
@@ -827,7 +827,7 @@
...
 	"You are trying to run a windows .exe binary inside WSL. For better integration please use a Linux binary instead (Download at https://minikube.sigs.k8s.io/docs/start/.). Otherwise if you still want to do this, you can do it using --force": "WSL 内で Windows の .exe バイナリーを実行しようとしています。これより優れた統合として、Linux バイナリーを代わりに使用してください (https://minikube.sigs.k8s.io/docs/start/ でダウンロードしてください)。そうではなく、引き続きこのバイナリーを使用したい場合、--force オプションを使用してください",
 	"You are trying to run the amd64 binary on an M1 system.\nPlease consider running the darwin/arm64 binary instead.\nDownload at {{.url}}": "M1 システム上で amd64 バイナリーを実行しようとしています。\ndarwin/arm64 バイナリーを代わりに実行することをご検討ください。\n{{.url}} でダウンロードしてください。",
-	"You are using the QEMU driver without a dedicated network, which doesn't support `minikube service` \u0026 `minikube tunnel` commands.": "",
+	"You are using the QEMU driver without a dedicated network, which doesn't support `minikube service` \u0026 `minikube tunnel` commands.": "独立したネットワークなしで QEMU ドライバーを使用しています。これは`minikube service` \u0026 `minikube tunnel` コマンドをサポートしていません。",
```
[the remark this answers is not in the document]
Comment (YuikoTakada) [reply translations/ja.json:830]: done

File: translations/ja.json [hunk 6]
```diff
@@ -913,7 +913,7 @@
...
 	"minikube is unable to connect to the VM: {{.error}}\n\n\tThis is likely due to one of two reasons:\n\n\t- VPN or firewall interference\n\t- {{.hypervisor}} network configuration issue\n\n\tSuggested workarounds:\n\n\t- Disable your local VPN or firewall software\n\t- Configure your local VPN or firewall to allow access to {{.ip}}\n\t- Restart or reinstall {{.hypervisor}}\n\t- Use an alternative --vm-driver\n\t- Use --force to override this connectivity check\n\t": "minikube が VM に接続できません: {{.error}}\n\n\t考えられる理由は以下の 2 つです:\n\n\t- VPN またはファイアウォールによる干渉\n\t- {{.hypervisor}} のネットワーク設定の問題\n\n\t回避策には以下があります:\n\n\t- ローカルの VPN またはファイアウォールを無効化\n\t- {{.ip}} へのアクセスを許可するようにローカルの VPN またはファイアウォールを設定\n\t- {{.hypervisor}} を再起動または再インストール\n\t- 代わりの --vm-driver を使用\n\t- --force を使用してこの接続チェックを上書き\n\t",
-	"minikube mount is not currently implemented with the user network on QEMU": "",
+	"minikube mount is not currently implemented with the user network on QEMU": "minikube トンネルは、QEMU 上のユーザーネットワークでは実装されていません。",
```
[the remark this answers is not in the document]
Comment (YuikoTakada) [reply translations/ja.json:916]: done

**Review by YuikoTakada — COMMENTED** [review]
@t-inu Thank you for your comments. I've fixed them, PTAL

**Review by t-inu — APPROVED** [review]
I've confirmed the fixes were done correctly.

**Comment by atoato88** [comment]
@YuikoTakada 
We can remove "[WIP]" from titile! :+1:

**Pull request merged**